### ISSUE TITLE
Line Issue When Formatting Comments/Summaries

### ISSUE THREAD

**Issue by KujanL** [genome/civic-client#725]
When leaving a comment or revision summary if I start a new sentence on the next line, when submitted it automatically moves up to the previous line. 

I demonstrated this when typing "Adding Ensembl version of HGVS expression for this variant" at the end. 

Screenshot of submission before submitting:
<img width="835" alt="screen shot 2017-04-05 at 5 05 35 pm" src="https://cloud.githubusercontent.com/assets/15361912/24729138/72ac34b4-1a22-11e7-87d7-1edd0bd94879.png">

Screenshot of submission once submitted:
<img width="953" alt="screen shot 2017-04-05 at 5 05 58 pm" src="https://cloud.githubusercontent.com/assets/15361912/24729164/84f0f952-1a22-11e7-8d37-d6eeb8231950.png">

Basically, I have to always go down two lines to start a new line (press 'Enter' twice).

**Comment by jmcmichael** [comment]
The CIViC server parses comments with a [Markdown formatter](https://en.wikipedia.org/wiki/Markdown) to generate the HTML used by browsers to actually display the comment, thus enabling links, @mentions, styling etc.

Our parser appears to render soft line breaks properly according to [the Markdown specification for soft breaks](http://spec.commonmark.org/0.27/#soft-line-breaks). 

[According to the spec for hard line breaks](http://spec.commonmark.org/0.27/#hard-line-breaks), unless a line ends with multiple spaces, a backslash character, or two blank lines, then the parser will simply render a space and/or end-of-line character (and the end-of-line character will be rendered by browsers as a space, unless it's in `<pre>` tags).

However, I tried to force a hard break with multiple spaces, and with a backslash and neither was parsed as a hard line break by the Markdown parser. So it appears the parser we're using isn't parsing hard line breaks according to spec. @acoffman - does the parser provide config options where we can tweak its settings to honor multiple spaces and/or backslashes to force hard line breaks?

**Comment by kkrysiak** [comment]
@acoffman  see question above
